src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -46,28 +46,25 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.anon_primary
   },
   textContainer: {
-    paddingLeft: BUBBLE_LEFT_PADDING,
+    paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
     paddingRight: BUBBLE_RIGHT_PADDING,
     paddingTop: 4,
     paddingBottom: 4,
-    borderRadius: 8,
+    borderRadius: 12,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The border radius has been increased from 8 to 12. This will make the corners more rounded. Ensure that this change aligns with your design requirements.



<!-- This is an auto-generated comment by OSS CodeRabbit -->